### ISSUE TITLE
Fix one color

### ISSUE THREAD

**Issue by ngargg** [busyloop/lolcat#118]
Hi there!

Firstly, THANK YOU for making such a beautiful software.

My doubt is can we make lolcat to have fixed colour of our choice, instead of picking up random colours?
Actually, I like one particular combination. So it would be nice enough to see only that one.

Thanks again

**Comment by WHK102** [comment]
Yeah, by example, use only lighting colours, like as blue, yellow, green, and white. The pink and red color is very dark.

**Comment by m-o-e** [comment]
Try `cal | lolcat -S 123`.

With the number after `-S` you pick a specific rainbow. Otherwise lolcat picks at random.

**Comment by yhojann-cl** [comment]
`ls -lah | lolcat -S 123` show a dark colors like as violet but need only light green, yellow and light blue.

**Comment by m-o-e** [comment]
Try other numbers than `123`, too. 🪄

**Comment by silvia552** [comment]
> `ls -lah | lolcat -S 123` show a dark colors like as violet but need only light green, yellow and light blue.

lolcat doesnt have an option to specify a color range or even a single color in its current state, you'll have to go through random numbers with `-S` until you find one that is closest to what you want
